Modules/Office365DSC/Modules/O365DSCReverseGUI.psm1

```diff
@@ -565,6 +565,14 @@ function Show-O365GUI
         $chckSCRetentionComplianceRule.Text = "Retention Compliance Rule"
         $pnlSC.Controls.Add($chckSCRetentionComplianceRule)
 
+        $chckSCSensitivityLabel = New-Object System.Windows.Forms.CheckBox
+        $chckSCSensitivityLabel.Top = 200
```

Comment:
Pretty sure the Top property is not right. If the next one is 320, this one should probably be 300 (we allow 20 px per item). In the future, no need to handle the ReverseDSC cases, I am more than happy to do it in a separate PR. YOu folks should just focus on the 4 methods: Get, Set, Test and Export